resources/views/auth/register.blade.php

```diff
@@ -0,0 +1,185 @@
+                                        </span>
+                                        @endif
+                                    </div>
+
+                                </div>
+                                <div class="col-sm-6 col-xs-12 mb-20">
+                                    <label for="r_email">Email Address <span class="required">*</span></label>
+                                    <input id="r_email" type="email"
+                                           name="email" value="{{ old('email') }}" >
+
+                                    @if ($errors->has('email'))
+                                        <span class="invalid-feedback" role="alert">
+                                            <strong>{{ $errors->first('email') }}</strong>
+                                        </span>
+                                    @endif
+                                </div>
+
+                                <div class="col-sm-6 col-xs-12 mb-20">
+                                    <label for="r_phone">Phone  <span class="required">*</span></label>
+                                    <input id="r_phone" type="text"
+                                           name="phone" value="{{ old('phone') }}"/>
+
+                                    @if ($errors->has('phone'))
+                                        <span class="invalid-feedback" role="alert">
+                                            <strong>{{ $errors->first('phone') }}</strong>
+                                        </span>
+                                    @endif
+                                </div>
+
+                                <div class="col-xs-12 mb-20">
+                                    <label for="r_country">Country <span class="required">*</span></label>
+                                    <select name="country" id="r_country">
+                                        <option value="1">Select a country</option>
+                                        <option value="2">bangladesh</option>
+                                        <option value="3">Algeria</option>
+                                        <option value="4">Afghanistan</option>
+                                        <option value="5">Ghana</option>
+                                        <option value="6">Albania</option>
+                                        <option value="7">Bahrain</option>
+                                        <option value="8">Colombia</option>
+                                        <option value="9">Dominican Republic</option>
+                                    </select>
+
+                                    @if ($errors->has('country'))
+                                        <span class="invalid-feedback" role="alert">
+                                            <strong>{{ $errors->first('country') }}</strong>
+                                        </span>
```

Comment:
Overall looks ok except for the issue mentioned above. Testing this in a bit

Reply:
ok